package.json

```diff
@@ -122,6 +123,6 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": ">=12"
+    "node": ">=16.15.1"
```

Comment:
The rollup deps did not work with Node 14, even with 16 I needed to pin to an older version of rollup.

Given 16 is already EOL, and 14 and 12 have been EOL for quite some time, I suggest we just use 16 as the base node engine.

Reply:
(I found trying to get a build going with Node.js 14 in Yargs libraries pretty painful.)